zeppelin-web/src/app/tabledata/tabledata.js

```diff
@@ -59,6 +59,9 @@ export default class TableData extends Dataset {
         if (i === 0) {
           columnNames.push({name: col, index: j, aggr: 'sum'});
         } else {
+          if (col.match('^[0-9]+([.,][0-9]+)?$')) {
```

Comment:
negative number?

Reply:
instead of applying a regex, is there a js function to try parse the string as float?